### PR TITLE
Support wide datasets (#93)

### PR DESCRIPTION
**Pull request by CanyonFoot**
Add support for wide datasets using row_keys argument when geographic keys are insufficient.

### CODE REVIEW

**Comment by vengroff** [comment]
I think it looks good. The one thing I would like to see before merging is a unit test. Basically you can add one to `test_integration.py` that attempts to do the download that failed and inspired this issue. It should work now, and by making it a unit test we will ensure that it does not regress with future changes.

File: reports/coverage/coverage-badge.svg
Comment (vengroff) [inline reports/coverage/coverage-badge.svg:1]: Adding the test will also keep this number from falling.

**Comment by vengroff** [comment]
I pushed a small change to deal with flake8 and black disagreeing. That was causing the lint action to fail but it should work now.

**Comment by CanyonFoot** [comment]
The unit test is in - lmk if there is anything else!

**Pull request merged**